vector/src/main/res/values/strings_riotX.xml

```diff
@@ -22,4 +22,6 @@
     <string name="a11y_create_room">Create a new room</string>
     <string name="a11y_close_keys_backup_banner">Close keys backup banner</string>
 
+    <string name="error_file_too_big">The file %1$s is too large to upload. The file size limit is %2$s but this file is %3$s.</string>
```

Comment:
I think something like: `The file 'cat.mp4' (50 MB) is too large to upload. The limit is 10 MB.` is more concise, but I guess this is a matter of opinion.

Reply:
I've picked the [Riot-Web](https://github.com/matrix-org/matrix-react-sdk/blob/develop/src/i18n/strings/en_EN.json#L1400) string, but yes, your wording is better, thanks